### ISSUE TITLE
albertv2有更新的打算吗？

### ISSUE THREAD

**Issue by brealisty** [brightmart/albert_zh#95]
大神，有没有更新v2版本的打算？

**Comment by brightmart** [comment]
v2我看到的是这样的：
In this version, we apply 'no dropout', 'additional training data' and 'long training time' strategies to all models

我们推出的时候就是'no dropout', 'long training time', 已经使用30G的训练数据，不过口语化的数据，还可以再加一些。

v2版本还有啥呢？

**Comment by brealisty** [comment]
> v2我看到的是这样的：
> In this version, we apply 'no dropout', 'additional training data' and 'long training time' strategies to all models
> 
> 我们推出的时候就是'no dropout', 'long training time', 已经使用30G的训练数据，不过口语化的数据，还可以再加一些。
> 
> v2版本还有啥呢？

嗯，介绍里确实也就只有这些了
